Stefan/src/main/java/ReferenceDataTypes.java

```diff
@@ -0,0 +1,14 @@
+public class ReferenceDataTypes {
+    public static void main(String[] args) {
+       validateRegex("2020-12-20");
+    }
+
+    private static void validateRegex(String date) {
```

Comment:
In this method you are validating the date, but not regex.
validateDate would sound more meaningful.

Reply:
Done

Reply:
Done